script.js

```diff
@@ -0,0 +1,102 @@
+// const API_URL = 'https://bible-api.com/?random=verse';
+const API_URL = 'https://bible-api.com/'; //Book + Chapter : Verse
+
+// URL Encoding Functions
+// encodeURLComponent ie. #
+
+async function getRandomBible(){
+    try{
+        const response = await fetch(API_URL + '?random=verse');
+        if (!response.ok){  //All of the fetches (API calls) should have a catch
+            throw new Error('Something wrong with getting the Bible' + error);
+        }
+        const data = await response.json();
+        return data;
+    } catch(error){
+        console.error('Something wrong with getting the Bible', error);
+    }
+}
+
+async function getBible(verse){
+    try{
+        const response = await fetch(API_URL + 'john ' + verse);
+        if (!response.ok){  //All of the fetches (API calls) should have a catch
+            throw new Error('Something wrong with getting the Bible' + error);
+        }
+        const data = await response.json();
+        return data;
+    } catch(error){
+        console.error('Something wrong with getting the Bible', error);
+    }
+}
+
+const prayButton = document.querySelector('.prayButton');
+const userSection = document.querySelector('.userInfo');
+const holySection = document.querySelector('.holyInfo');
+const verse = document.querySelector('.verse');
+const text = document.querySelector('.text');
+const dailyText = document.querySelector('.dailyText');
+
+let userVerse = "";
+
+holySection.style.display = 'none';
+
+function buttonClick(){
+    if (holySection.style.display == 'none'){
+        userSection.style.display = 'none';
+        holySection.style.display = 'flex';
+        prayButton.innerHTML = '✞ Try Another Name ✞';
+    }else{
+        userSection.style.display = 'flex';
+        holySection.style.display = 'none';
+        prayButton.innerHTML = '✞ Amen ✞';
+    }
+}
+
+
+function holyCalc(){
+    let input = document.querySelectorAll('.user input');
+    let name = '';
+    let year = '';
+    for (let [index, inputElement] of input.entries()) {
+        if (index == 0){
+        name = inputElement.value;
+        }else{
+        year = inputElement.value;
+        }
+    }
+    holyVerse(name,year)
+}
+
+function holyVerse(name, year){
+    let chpt = name.length;
+    let vrs = 1
+    if(chpt >= 12){
+        chpt = name.length/2
+    }
+    if(year%2 == 0){
+        vrs = chpt*2
+    }else{
+        vrs = chpt+2
+    }
+    userVerse = `${chpt}:${vrs}`
+}
+
+async function renderHoliness(){
+    const bibleText = await getBible(userVerse)
+
+    verse.innerText = bibleText["verses"][0]["book_name"] + ' ' + bibleText["verses"][0]["chapter"] + ':' + bibleText["verses"][0]["verse"];
```

Comment:
Just an FYI: there may be a case where `bibleText["verses][0]` does not exist. If that happens, then your code will through an error and break your page. When it comes to selecting properties within properties within properties etc...you should wrap them in an if statement so that it will not throw an error. In this case, you should do:
```
if (bibleText["verses"][0]) {
  verse.innerText = bibleText["verses"][0]["book_name"] + ' ' + bibleText["verses"][0]["chapter"] + ':' + bibleText["verses"][0]["verse"];
}
```